admin_manual/release_notes.rst

```diff
@@ -12,6 +13,65 @@ Release Notes
+
+- **Added mode of operations to differentiate between single-instance or clustered setup**
+As ownCloud needs to behave differently when operating in a clustered setup versus a single instance setup, the new `config.php` option `operation.mode` has been added. It can take one of two values: `single-instance` and `clustered-instance`.
+For example:
+
+```
+'operation.mode' => 'clustered-instance',
+```
+
+Currently the Market App (ownCloud Marketplace integration) does not support clustered setups and can do harm when used for installing or updating apps. The new config setting prevents this and other actions that are undesired in cluster mode.
+**Therefore when operating in a clustered setup it is mandatory to set this option.**
+Please check the ownCloud documentation for more information (https://doc.owncloud.com/server/10.0/admin_manual/configuration/server/config_sample_php_parameters.html#mode-of-operation).
+
+- **Added occ dav:cleanup-chunks command to clean up expired uploads**
+When file uploads are interrupted for any reason, already uploaded file parts (chunks) remain in the underlying storage so that the file upload can resume in a future upload attempt.
+However, resuming an upload is only possible until the partial upload is expired and deleted, respectively. To clean up chunks (expire and delete) originating from unfinished uploads, administrators can use this newly introduced command. The default expiry time is two days, but it can be specified as a parameter to the command.
+**It is recommended to configure CRON to execute this background job regularly**. It is not included in the regular ownCloud background jobs so that the administrators have more flexibility in scheduling it. Please check the ownCloud documentation for more information (https://doc.owncloud.com/server/10.0/admin_manual/configuration/server/background_jobs_configuration.html#cleanupchunks).
+
+- **Administrators can now exclude files from integrity check in config.php**
+When administrators did intentional changes to the ownCloud code they now have the ability to exclude certain files from the integrity checker.
+Please check "config.sample.php" for the usage of ``'integrity.excluded.files'``.
+
+- **Modification time value of files is now 64 bits long**
+Note: When upgrading to 10.0.4 migrations may increase update duration dependent on number of files.
+
+- **Updated minimum supported browser versions**
+Note: Users with outdated browsers might get warnings. See the list of supported browser versions: https://doc.owncloud.com/server/10.0/admin_manual/installation/system_requirements.html#web-browser
+
+- **10.0.3 known issues are solved**
+  - SFTP external storages with key pair mode work again
+  - Added support for MariaDB 10.2.7+
+  - Encryption panel in admin settings fixed to properly detect current mode after upgrade to ownCloud 10
+
+**Known issues**
+
+Currently there are no known issues for ownCloud Server 10.0.4.
```

Comment:
Should we remove this section ? This seems to state that ownCloud 10.0.4 is bug-free which is not correct 😄

Reply:
As no software's bug-free, I'd say yes.

Reply:
As I said above I left the section not to confuse people that we might have missed to add known issues. This way it states that there are no issues that we consider as "known" and "to-be-fixed". Should we remove it?

Reply:
No, I'd leave it. It's clear in what it says, and it's honest.